src/EnergyPlus/HVACFourPipeBeam.cc

```diff
@@ -100,16 +100,10 @@ namespace FourPipeBeam {
 
         int beamIndex; // loop index
 
-        int NumAlphas(0);  // Number of Alphas for each GetObjectItem call
-        int NumNumbers(0); // Number of Numbers for each GetObjectItem call
-
         //  certain object in the input file
-        int IOStatus; // Used in GetObjectItem
         bool errFlag = false;
         bool ErrorsFound(false); // Set to true if errors in input, fatal at end of routine
         bool found = false;
-        int ctrlZone; // controlled zome do loop index
-        int supAirIn; // controlled zone supply air inlet index
```

Comment:
[src/EnergyPlus/HVACFourPipeBeam.cc:107]:(style),[variableScope],The scope of the variable 'IOStatus' can be reduced.
[src/EnergyPlus/HVACFourPipeBeam.cc:111]:(style),[variableScope],The scope of the variable 'ctrlZone' can be reduced.
[src/EnergyPlus/HVACFourPipeBeam.cc:112]:(style),[variableScope],The scope of the variable 'supAirIn' can be reduced.